app/src/main/java/org/systers/mentorship/view/fragments/ProfileFragment.kt

```diff
@@ -63,7 +63,7 @@ class ProfileFragment : BaseFragment() {
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
         return when (item.itemId) {
             R.id.menu_edit_profile -> {
-                EditProfileFragment.newInstance(profileViewModel.user).show(fragmentManager,
+                EditProfileFragment.newInstance(profileViewModel.user).show(fragmentManager!!,
```

Comment:
What is the use of adding '!!' here?

Reply:
Because <fragment_name>.newInstance(<view_model>).show() has first argument as FragmentManager but what we orginally passed was FragmentManager? so I had to assert FragmentManager? using !!